app/views/transfers/_transfer_row.html.erb

```diff
@@ -5,7 +5,9 @@
   <td><%= transfer_row.comment || "none" %></td>
   <td><%= transfer_row.line_items.total %></td>
   <td>
-    <%= delete_button_to transfer_path(transfer_row.id), { confirm: "Are you sure you want to permanently remove this transfer?" } %>
+    <% if transfer_row.deletable? %>
```

Comment:
In general we don't do validations before we perform the action. @cielf would it be OK to only have this raise an error and not try to change the view beforehand? I think this is an edge case and I don't want to overthink this.

Reply:
It's *definitely* an edge case.  I'm ok with that -- we need to be pretty clear on why they can't in the error, though.